modules/nf-core/yak/count/tests/main.nf.test

```diff
@@ -0,0 +1,59 @@
+nextflow_process {
+
+    name "Test Process YAK_COUNT"
+    script "../main.nf"
+    process "YAK_COUNT"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "yak"
+    tag "yak/count"
+
+    test("test_yak_count_methylated_1_fastq_gz") {
+
+        when {
+            process {
+                """                
+                input[0] = [
+                    [ id:'test' ], // meta map
+                    file(params.test_data['sarscov2']['illumina']['test_methylated_1_fastq_gz'], checkIfExists: true)
+                ]
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out).match() }
+            )
+        }
+
+    }
+
+    // TODO nf-core: Change the test name preferably indicating the test-data and file-format used but keep the " - stub" suffix.
+    test("test_yak_count_methylated_1_fastq_gz - stub") {
```

Comment:
- remove todo statement
- add pe test
- update test name

Reply:
you are right, I was thinking of having a step to concatenate the two fastqs before the yak, amazing solution.

I will incorporate your suggestion and resubmit :)

Thank you